packages/amplify-e2e-core/src/utils/sdk-calls.ts

```diff
@@ -98,15 +115,13 @@ export const deleteS3Bucket = async (bucket: string, providedS3Client: S3 | unde
   } while (truncated);
   const chunkedResult = _.chunk(objectKeyAndVersion, 1000);
   const deleteReq = chunkedResult
-    .map(r => {
-      return {
-        Bucket: bucket,
-        Delete: {
-          Objects: r,
-          Quiet: true,
-        },
-      };
-    })
+    .map(r => ({
+      Bucket: bucket,
+      Delete: {
+        Objects: r,
+        Quiet: true,
+      },
+    }))
```

Comment:
```suggestion
    }));
```

Reply:
This expression continues with the `.map` below